src/bkl/props.py

```diff
@@ -92,8 +92,6 @@ def std_file_props():
              doc="""
                  List of additional files that the commands that compiles this
                  source file depend on.
-
-                 Only applicable if *compile-commands* is set.
```

Comment:
The documentation remains inaccurate even with this removal, because it now reads "List of additional files that the commands that compiles this source file depend on.” — there’s no “command” anymore, so it should be clarified that the dependencies are either for the command or simply dependencies on other (generated) files.